tests/browser/test_shuttle_assignment.py

```
@@ -275,3 +275,196 @@ def test_assignment_summary_statistics(
         page_source = driver.page_source
         assert ">Proj<" in page_source
         assert ">Slots<" in page_source
+
+    def test_projects_table_shows_owner_column(
+        self, driver, wait, staff_user, shuttle, project_with_compliance
```

Ruff (0.14.7)

Unused method argument: `project_with_compliance`

(ARG002)